model/client4.go

```diff
@@ -3100,6 +3100,24 @@ func (c *Client4) GetChannelsForTeamAndUserWithLastDeleteAt(teamId, userId strin
 	return ch, BuildResponse(r), nil
 }
 
+// GetChannelsForUserWithLastDeleteAt returns a list channels for a user, additionally filtered with lastDeleteAt.
+func (c *Client4) GetChannelsForUserWithLastDeleteAt(userID string, lastDeleteAt int) ([]*Channel, *Response, error) {
```

Comment:
I was trying to match with the other existing methods. But 0/5, let me know.

Reply:
It feels like `lastDeleteAt` is really a filtering option so from an API design perspective I think it would be wiser to have it inside an object that can be extended without breaking things nor requiring adding yet another method. That said, if you think this method is not going to need any addition in the foreseeable future (v6 lifespan) then it's totally fine to leave it as is.